DEVELOPMENT.md

```diff
@@ -0,0 +1,93 @@
+## Table of content
+
+- [Developing package](#developing-package)
+- [Developing website](#developing-website)
+
+# Developing package
+
+`/modules/manifold` contains the content of Manifold package, and `/example/manifold` is the playground where you can develop the package.
+
+## Run example locally
+
+In root directory:
+
+```sh
+yarn install
+```
+
+In `/example/manifold` directory:
+
+```sh
+yarn install
+yarn start
+```
+
+All changes made in `/modules/manifold` or other packages that are used in Manifold (e.g. `/modules/feature-list-view`) will be hot-loaded while you are running `/example/manifold` locally.
+
+## Deploy package
+
+After finishing developing the package and after merging the pull request into `master`, you can deploy it to on npm.
+
+In root directory, on `master` branch:
+
+```sh
+yarn bump
+```
+
+Then choose the version number you'd like to deploy.
+
+Still in root directory:
+
+```sh
+git push origin master
+```
+
+And travis CI will automatically deploy the newest version to npm.
```

Comment:
Not so sure on how Travis CI determines when to deploy. Does it check the version number on package.json of each module to determine whether to deploy on the push to master.

Reply:
See above, I will update the documentation to make it more clear.